src/renderer/store/certificates/certificates.saga.ts

```diff
@@ -15,28 +15,39 @@ export function* responseGetCertificates(
   yield put(certificatesActions.setUsersCertificates(certificates))
 }
 
-const getDate = (date?: string) => new Date(date)
+export const getDate = (date?: string) => date ? new Date(date) : new Date()
 
 export function* creactOwnCertificate(
   action: PayloadAction<ReturnType<typeof certificatesActions.creactOwnCertificate>['payload']>
 ): Generator {
   const certString = dataFromRootPems.certificate
   const keyString = dataFromRootPems.privKey
-
   const notBeforeDate = yield call(getDate)
   const notAfterDate = yield call(getDate, '1/1/2031')
 
-  yield apply(electronStore, electronStore.get, ['hiddenServices'])
+  interface HiddenServicesType {
+    libp2pHiddenService?: {
+      onionAddress: string
+      privateKey: string
+    }
+  }
+
+  const hiddenServices: HiddenServicesType = yield apply(
+    electronStore,
+    electronStore.get,
+    ['hiddenServices']
```

Comment:
I think it should be 
```

 const hiddenServices: HiddenServicesType = yield apply(    electronStore,    electronStore.get,    'hiddenServices'  )
 
 ```
 
 (without the wrapping in list)

Reply:
![image](https://user-images.githubusercontent.com/61462655/122567737-4a241200-d049-11eb-944a-992002dce4a1.png)

Reply:
i found it in the documentation

Reply:
Ok, ok than.